app/domain/authentication/authn_oidc/authenticate_id_token/authenticate.rb

```diff
@@ -92,12 +92,12 @@ def validate_security
               user_id: @authenticator_input.username,
               enabled_authenticators: @enabled_authenticators
           )
-          @logger.debug("[OIDC] Security validated")
+          @logger.debug(::LogMessages::Authentication::Security::SecurityValidated.new.to_s)
```

Comment:
Check if you need `to_s` here.  A lot of times (but not always) methods like this will auto-call to `to_s` when you pass them an object.

Reply:
checked it. without the `new.to_s` it prints `::LogMessages::Authentication::Security::SecurityValidated`